### DISCUSSION TITLE
Supprimer téléchargement répertoire pour les apprenants Chamilo 1.11 26

### DISCUSSION THREAD

**Discussion by Mounaim33** [chamilo/chamilo-lms#5522] [General]
Bonjour,
je n'arrive pas à supprimer le téléchargement des répertoires dans l'outil documents pour les apprenants .
malgré que j'ai interdit l'option . cf . copie écran.
Merci pour votre aide .
![Screenshot 2024-05-18 at 11-39-03 EDUCADIS - educadis - Paramètres de configuration de Chamilo](https://github.com/chamilo/chamilo-lms/assets/50979940/004c4796-a6d0-4486-bea7-eb9035ad01d9)

**Comment by ktgbf84** [comment]
comment a tu fais pour activé l'option copie d'écran
